jetty-core/jetty-deploy/src/main/config/etc/jetty-deployment-scanner.xml

```diff
@@ -15,14 +15,14 @@
       <New id="deploymentScanner" class="org.eclipse.jetty.deploy.DeploymentScanner">
         <Arg name="server"><Ref refid="Server"/></Arg>
         <Arg name="deployer"><Ref refid="Deployer"/></Arg>
-        <Call name="addMonitoredDirectory">
+        <Call name="addWebappsDirectory">
```

Comment:
Should we call the setter and have the property be a CSV list?

Reply:
This PR is about renaming, not new features. (scope creep)
If you want to explore CVS and/or list support, lets do that in a different PR.